prov/verbs/src/fi_verbs.h

```diff
@@ -285,6 +313,16 @@ typedef int(*fi_ibv_mr_dereg_cb)(struct fi_ibv_mem_desc *md);
 
 struct fi_ibv_mem_notifier;
 
+struct fi_ibv_xrc_domain {
+	int				xrcd_fd;
+	struct ibv_xrcd			*xrcd;
+
+	/* The domain maintains a RBTree for mapping an endpoint destination
+	 * addresses to physical XRC INI QP connected to that host. */
+	fastlock_t			ini_mgmt_lock;
+	struct ofi_rbmap		*ini_conn_rbmap;
+};
```

Comment:
Does this need to be a separate structure, or can we embed the struct directly into fi_ibv_domain?

Reply:
Thanks, yes it makes sense for definition to be embedded in the fi_ibv_domain.

Reply:
Thanks, I embedded the structure in the fi_ibv_domain.